obspy/core/event/event.py

```diff
@@ -0,0 +1,379 @@
+        :class:`~obspy.core.event.StationMagnitude`
+    :param station_magnitudes: Station magnitudes associated with the event.
+
+    .. note::
+
+        For handling additional information not covered by the QuakeML
+        standard and how to output it to QuakeML see the
+        :ref:`ObsPy Tutorial <quakeml-extra>`.
+    """
+    def short_str(self):
+        """
+        Returns a short string representation of the current Event.
+
+        Example:
+        Time | Lat | Long | Magnitude of the first origin, e.g.
+        2011-03-11T05:46:24.120000Z | +38.297, +142.373 | 9.1 MW
+        """
+        out = ''
+        origin = None
+        if self.origins:
+            origin = self.preferred_origin() or self.origins[0]
+            out += '%s | %+7.3f, %+8.3f' % (origin.time,
+                                            origin.latitude,
+                                            origin.longitude)
+        if self.magnitudes:
+            magnitude = self.preferred_magnitude() or self.magnitudes[0]
+            out += ' | %s %-2s' % (magnitude.mag,
+                                   magnitude.magnitude_type)
+        if origin and origin.evaluation_mode:
+            out += ' | %s' % (origin.evaluation_mode)
+        return out
+
+    def __str__(self):
+        """
+        Print a short summary at the top.
+        """
+        return "Event:\t%s\n\n%s" % (
+            self.short_str(),
+            "\n".join(super(Event, self).__str__().split("\n")[1:]))
+
+    def _repr_pretty_(self, p, cycle):
+        p.text(str(self))
+
+    def __repr__(self):
+        return super(Event, self).__str__(force_one_line=True)
+
+    def preferred_origin(self):
+        """
+        Returns the preferred origin
+        """
+        try:
+            return ResourceIdentifier(self.preferred_origin_id).\
+                get_referred_object()
+        except AttributeError:
+            return None
+
+    def preferred_magnitude(self):
+        """
+        Returns the preferred magnitude
+        """
+        try:
+            return ResourceIdentifier(self.preferred_magnitude_id).\
+                get_referred_object()
+        except AttributeError:
+            return None
+
+    def preferred_focal_mechanism(self):
+        """
+        Returns the preferred focal mechanism
+        """
+        try:
+            return ResourceIdentifier(self.preferred_focal_mechanism_id).\
+                get_referred_object()
+        except AttributeError:
+            return None
+
+    def plot(self, kind=[['ortho', 'beachball'], ['p_sphere', 's_sphere']],
+             subplot_size=4.0, show=True, outfile=None, **kwargs):
+        """
+        Plot event location and/or the preferred focal mechanism
+        and radiation pattern.
+
+        :type kind: list of str or nested list of str
+        :param kind: can be one of the following options:
+            * A list of strings (for a 1-row plot) or a nested list
```

Comment:
I'm not sure why the bullet list is on this line instead of on the string options below.